docs/components/board/jetson.md

```diff
@@ -15,6 +15,12 @@ Follow one of our Jetson [setup guides](/installation/) to prepare your board fo
 
 {{% /alert %}}
 
+{{% alert title="REQUIREMENTS" color="warning" %}}
+
+The jetson's gpio pins are rated for inputs and outputs at 3.3V. Signals from encoders and sensors at even 5V can cause damage to a pin. We recommend connecting hardware that can operate and send signals at 3.3V or lower. [Jetson Nano Developer Kit 40-Pin Expansion Header GPIO Usage Considerations Applications Note](https://developer.nvidia.com/jetson-nano-developer-kit-40-pin-expansion-header-gpio-usage-considerations-applications-note)
```

Comment:
At the beginning of the paragraph, gpio is not capitalized, but at the end, GPIO is. Change one or the other so it's consistent.

Reply:
I clicked the link, and NVidia prompted me to create an account and log in before I could read whatever it is. After creating an account, I was shown a 12-page PDF, with no obvious indication of which part of the PDF you're trying to refer to. Is this link worth having at all?

Reply:
If you're going to keep the link, consider rephrasing it into a sentence: `For details, see page XX of the [Jetson Nano Developer Kit...](...)`

Reply:
I did want to ask @npentrel about including the page number. Seeed studio uploaded a version of this that you can access without an account, but I didn't want to link the unofficial documentation. I think the link is worth having if people want to read about other gpio considerations.

Reply:
```suggestion
The jetson's GPIO pins are rated for inputs and outputs at 3.3V. Signals from encoders and sensors at even 5V can cause damage to a pin. We recommend connecting hardware that can operate and send signals at 3.3V or lower. For details, see pages 1-3 of the [Jetson Nano Developer Kit 40-Pin Expansion Header GPIO Usage Considerations Applications Note](https://developer.nvidia.com/jetson-nano-developer-kit-40-pin-expansion-header-gpio-usage-considerations-applications-note)
```